configure-buildfarm/tasks/main.yml

```diff
@@ -48,6 +48,19 @@
     - config
     - security
 
+- name: "Add SSH public key to the user configuration page automatically"
+  uri:
+    url: "{{ jenkins_route_protocol | default('https') }}://{{ route_output.stdout }}/user/{{ jenkins_username | default('admin') }}/configSubmit"
+    method: POST
+    force_basic_auth: true
```

Comment:
How would it work with oauth tokens?

Reply:
It would work if the token would be passed the same way as a password below (just tested it on our CI server).
But I guess the problem here is that user cannot specify the token before running the playbook because the Jenkins instance does not exist yet :)

Wdyt about mentioning that if user wants to use this task, it would work only if he already has the oauth token.

Reply:
Agreed, we can add a note in both readme file and aerogear website.